pkg/api/http_server.go

```diff
@@ -180,6 +180,7 @@ func (hs *HTTPServer) getListener() (net.Listener, error) {
 		}
 
 		// Make socket writable by group
+		// nolint:gosec
```

Comment:
Can we use `// #nosec G<code>` instead? That should silence as little as we can.

Reply:
@sakjur It's not from this PR.